driver/normalizer/normalizer.go

```diff
@@ -479,3 +479,50 @@ func (op splitUse) Construct(st *State, n nodes.Node) (nodes.Node, error) {
 	// TODO: add some info to join nodes back
 	return st.MustGetVar(op.vr)
 }
+
+type opAdd struct {
```

Comment:
This would be useful in the SDK! The Python driver also applies an offset (currently in the native driver).